### ISSUE TITLE
Gate higher risk roles behind server whitelist

### ISSUE THREAD

**Issue by redmushie** [echo-station/echo-station#6]
Currently we have HoS, Warden and Captain restricted as role-whitelist. Building off of #5, what I actually want is to just straight up lock roles behind the **server** whitelist altogether.

The point of this is to prevent randoms from picking higher-risk roles. This way we also don't have to rely on time requirements that were applied to Janitor, Med Intern, Sec Cadet (not exhaustive) that can be easily circumvented by just joining as passenger a few times

To summarize:
- Gate entirety of Security behind server whitelist
- Gate all of Command  behind server whitelist
- Consider which other minor roles to apply this to (Med Intern, Janitor maybe?)

**Comment by redmushie** [comment]
Fixed by https://github.com/echo-station/echo-station/commit/ca95fff372ae48ed0a419544ada0ea3f444c98b1
